File: tests/fixtures/rust-semantic-family/host/semantic-core/src/dormant.rs
fn used_helper(value: i32) -> i32 {
    value + 1
}

#[allow(dead_code)]
fn dormant_discount(value: i32) -> i32 {
    value / 2
}

#[allow(dead_code)]
unsafe fn unsafe_dormant() -> i32 {
    0
}

pub fn used_total(value: i32) -> i32 {
    used_helper(value)
}

pub fn reflection_decoy() -> &'static str {
    "dormant_discount"
}

macro_rules! generated_handler {
    ($name:ident) => {
        fn $name() -> i32 {
            7
        }
    };
}

generated_handler!(macro_owned);

pub fn macro_value() -> i32 {
    macro_owned()
}

File: tests/fixtures/swift-semantic-a3/host/Sources/SwiftA3Core/Dormant.swift
private func dormantDiscount(_ value: Int) -> Int {
  value + 7
}

private func reflectedHelper(_ value: Int) -> Int {
  value + 11
}

private func usedHelper(_ value: Int) -> Int {
  value * 2
}

public func activeTotal(_ value: Int) -> Int {
  usedHelper(value)
}

public let dormantReflectionDecoy = "reflectedHelper"
